.github/workflows/build-package-docs.yaml

```diff
@@ -55,6 +55,16 @@ jobs:
   build-package-docs:
     runs-on: ubuntu-latest
     steps:
+    - name: Log the workflow inputs
+      run: |
+        echo '## Deployment details :rocket:' >> "${GITHUB_STEP_SUMMARY}"
+        echo '| PARAMETER | VALUE |' >> "${GITHUB_STEP_SUMMARY}"
+        echo '| --- | --- |' >> "${GITHUB_STEP_SUMMARY}"
```

Comment:
My 2c is that a table is too much formatting. I would prefer something simpler, even though a table might look nice.

Reply:
From the perspective of using this down the road, I find it easier to scan a table to understand what the input parameters are when looking at a recent build/test build. I find it more scannable than a bullet list of items personally.

Reply:
Consider [this commit](https://github.com/oraNod/ansible-documentation/commit/a2886f5a415ce121fe6d2c378b9c93fa5033ca9d). It has less lines and less characters and results in a pretty clean summary section:

![image](https://github.com/ansible/ansible-documentation/assets/32749632/9190e799-9127-4cc6-b667-5706e803d784)

It's not a table but it does offer the deployment details "at a glance". I personally find that just as easy to scan.

Reply:
OK, I wanted to double check with the quotes but we can use a single set of double quotes like [this commit](https://github.com/oraNod/ansible-documentation/commit/7157eb478eaa397b41a983d2969ed6bfb559afd3) that should avoid any sort of concatenation errors like I mentioned before.

Reply:
agreed and pulled in your two commits... thanks!